src/components/CvCheckbox/__tests__/CvCheckbox.spec.js

```diff
@@ -0,0 +1,37 @@
+import { render, fireEvent } from '@testing-library/vue';
+import CvCheckbox from '../CvCheckbox';
+
+describe('CvCheckbox', () => {
+  it('CvCheckbox - default', async () => {
+    // The render method returns a collection of utilities to query your component.
+    const { emitted, getByTestId } = render(CvCheckbox, {
+      props: { value: 'checkbox-1' },
+    });
+
+    const checkbox = getByTestId('checkbox');
```

Comment:
I would ordinarily only use getByTestId as a last resort. In this case, and the next,  getByRole would be more appropriate

Reply:
`$listeners` combined into `attrs` in Vue3 of course. I added test for `@click` listener on the checkbox and set an additional attribute to test those.